pkg/sql/grant_revoke.go

```diff
@@ -58,6 +60,27 @@ func (p *planner) changePrivileges(
 	grantees tree.NameList,
 	changePrivilege func(*sqlbase.PrivilegeDescriptor, string),
 ) (planNode, error) {
+	//check whether grantees exists
+	users, err := GetAllUsers(ctx, p)
+
+	if err != nil {
+		return nil, err
+	}
+	users = append(users, security.RootUser)
+	users = append(users, security.NodeUser)
+	for _, grantee := range grantees {
```

Comment:
I would just return a `map[string]{}` of users. Then the inside of the loop becomes just `if _, ok := users[grantee]; !ok {`

Reply:
ok